src/assets/scss/settings/base/_colors.scss

```diff
@@ -13,8 +13,8 @@ $palette: (
     3: #33363a,
   ),
   primary: (
-    1: #2f90ff,
-    2: #99c7ff,
+    1: #1360b8,
```

Comment:
Made the color slightly darker so we passed an accessibility check with Google Lighthouse (AAA contrast)